administrator/language/en-GB/guidedtours.joomla_welcome_steps.ini

```
@@ -0,0 +1,14 @@
+COM_GUIDEDTOURS_TOUR_WELCOMETOJOOMLA_STEP_MENUS_TITLE="The menu"
+COM_GUIDEDTOURS_TOUR_WELCOMETOJOOMLA_STEP_MENUS_DESCRIPTION="<p>The menu provides access to the Administrator management pages. Each section has a collection of pages for managing that particular aspect of Joomla, such as Content, Menus and Users.</p><p>The window pane icons link to individual dashboards for those menu groups. The System menu itself leads to a dashboard.</p>"
+
+COM_GUIDEDTOURS_TOUR_WELCOMETOJOOMLA_STEP_QUICKACCESS_TITLE="The dashboard panels"
+COM_GUIDEDTOURS_TOUR_WELCOMETOJOOMLA_STEP_QUICKACCESS_DESCRIPTION="<p>Dashboards provide quick access to frequently used pages via their icons.</p><p>Each dashboard panel is an Administrator module that can be added to or removed from a dashboard and each panel may have items added or removed via its Settings icon <span class=\"icon-cogs\" aria-hidden=\"true\"></span>. This allows you to configure your site dashboards to suit your own purposes.</p>"
+
+COM_GUIDEDTOURS_TOUR_WELCOMETOJOOMLA_STEP_NOTIFICATIONS_TITLE="The Notifications panel"
+COM_GUIDEDTOURS_TOUR_WELCOMETOJOOMLA_STEP_NOTIFICATIONS_DESCRIPTION="<p>The Notifications panel shows whether updates are available for Joomla and other extensions that provide update sites. Any outstanding Privacy Requests are shown here too.</p>"
+
+COM_GUIDEDTOURS_TOUR_WELCOMETOJOOMLA_STEP_TOPBAR_TITLE="The top bar"
+COM_GUIDEDTOURS_TOUR_WELCOMETOJOOMLA_STEP_TOPBAR_DESCRIPTION="<p>The top bar shows the title of the displayed page. The other items in the top bar are modules that you can configure to display or not as you see fit.</p><p>Your current Joomla version and the list of tours are often the next items in the top bar.</p><p>The last items are usually a link to the Site frontend and the User Menu where you can change your password, personal settings and logout.</p>"
+
+COM_GUIDEDTOURS_TOUR_WELCOMETOJOOMLA_STEP_FINALWORDS_TITLE="Help and information"
```

```suggestion
COM_GUIDEDTOURS_TOUR_WELCOMETOJOOMLA_STEP_FINALWORDS_TITLE="Help and Information"
```

Follows the style guide

---

Thx